TurkishFilter/sections/general_extensions.txt

```diff
@@ -674,6 +674,7 @@ fullfilmizle.pw#$#div[data-pns-locked-message="movie_player"] { display: none !i
 guncelhdfilm.com#$##movie { display: block !important; }
 guncelhdfilm.com#$##movieoncereklam { display: none !important; }
 oyunu.com.tr#$##bir.oynama { display: none !important; }
+caracad.com#$#body { background: #222 !important; }
```

Comment:
Wrong place. You have to add it with task link much higher than it is now.
Btw, I don't see ad background without the rule.

Reply:
This rules has nothing to do with ads, just a minor tweak, gonna move there then

Reply:
Ok.